generate.pl

```diff
@@ -199,7 +199,7 @@ =head1 DESCRIPTION
 
 COPY *.patch /usr/src/perl/
 WORKDIR /usr/src/perl
-
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y update && apt-get -y upgrade
```

Comment:
IIRC we do not do upgrades here as it is the responsibility of the image user to do so, as well as to reduce a layer of change from the base `buildpack-deps` image.  Please see [the docker-library/official-images README](https://github.com/docker-library/official-images#contributing-to-the-standard-library) for the relevant bits.

That said, I'm working to update the image for 5.26.1 this week and this will pull in the latest base image updates as well.